docs/en/latest/admin-api.md

```diff
@@ -657,12 +659,54 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: How to proxy requests to upstream with HTTPS scheme.
```

Comment:
Example 2: How to proxy client request to `https` upstream service

Reply:
Updated.